pkg/action/env.container.go

```diff
@@ -223,6 +242,19 @@ func (c *containerEnv) Close() error {
 	return c.driver.Close()
 }
 
+func (c *containerEnv) ImageRemove(ctx context.Context, a *Action) {
```

Comment:
The method shouldn't be public. It's your helper method. Rename to `imageRemove`.
Return error to the caller, the error must be handled by it.